packages/base/lib/plugin.js

```diff
@@ -13,9 +13,12 @@ export default class BasePlugin {
   // }
   static newMethodMap = {};
 
-  constructor (pluginName) {
+  constructor (pluginName, pluginArgs = {}) {
```

Comment:
we could also add a universal opts argument, so in the future it won't be necessary to to add more arguments, but instead put all the new stuff into `opts`

Reply:
yeah, not a bad idea. then we could pass it in as `opts.cliArgs` or similar